### ISSUE TITLE
Synchronize with provider

### ISSUE THREAD

**Issue by nonfungibletunji** [qutheory/red-feedback#26]
Expected behavior:
Actual behavior:
URL: /servers
User ID: 8B869FF8-27BF-44A0-9629-B0D3D3A26854
Company ID: A95B900C-D04F-41DD-AEBF-C6944E2A0496

I destroyed a server on the digital Ocean side through the UI, but is still shows in the UI in vapor.red

**Comment by joscdk** [comment]
@stunjiturner Yes this is a known thing. The problem is Digital Ocean don't send us any information when changing the server through their system unfortunately.

We are planning on adding a "Synchronize with provider" under settings, so if the server changes at some point through Digital Ocean (Shutdown, scale, delete etc.) pressing this button will make the changes in Vapor Red
